### PR TITLE
remedial work

### PR DESCRIPTION
**Pull request by matt-oc**
#1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +19,26 @@
...
   "bugs": {
     "url": "https://linkedin.com/nherment/seneca-linkedin-auth/issues"
   },
   "homepage": "https://linkedin.com/nherment/seneca-linkedin-auth",
   "dependencies": {
-    "passport-linkedin": "0.1.3"
+    "passport-linkedin": "1.0.0"
   },
+  "devDependencies": {
+     "eslint-config-seneca": "1.1.x",
```
Comment (ckiss) [inline package.json:38]: also you need all these three plugins installed: 
eslint-config-seneca
eslint-plugin-standard
eslint-plugin-hapi

after you install them, please run the linter again, you might find more linting issues

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "linkedin auth plugin for seneca-auth",
   "main": "linkedin-auth.js",
   "scripts": {
-    "test": "mocha test --reporter spec"
+    "test": "mocha test --reporter spec",
+    "lint": "lab test -dL"
```
Comment (ckiss) [inline package.json:9]: add an annotate script here

**Comment by ckiss** [comment]
looks good @matthewoconnor-development. You can see there small comments that needs to be addressed before merge. Also you are missing the .eslintrc file, please add it and in extend just seneca in it.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["seneca"]
+}
```
Comment (mihaidma) [inline .eslintrc]: no newline at end of file

File: docs/public/stylesheets/normalize.css [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+   ========================================================================== */
+
+/*
+ * Remove most spacing between table cells.
+ */
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
```
Comment (mihaidma) [inline docs/public/stylesheets/normalize.css]: no newline at end of file

**Pull request merged**